src/test/java/data/FileSystemDataAcquisistionTest.java

```diff
@@ -0,0 +1,44 @@
+package data;
+
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Timotej on 21-Oct-15.
+ */
+public class FileSystemDataAcquisistionTest {
+
+    FileSystemDataBase fsdb = new FileSystemDataBase("exampleData");
+    List<SubjectRawData> sd = fsdb.getSubjects();
```

Comment:
Maybe make these final so that they don't break tests in the future? Doesn't really matter. Also I would change name of FileSystemDataBase to FileSystemDatabase and any relevant variable names